.github/libs/GithubUtils.ts

```diff
@@ -157,144 +201,142 @@ class GithubUtils {
      * Parse the PRList and Internal QA section of the StagingDeployCash issue body.
      *
      * @private
-     *
-     * @param {Object} issue
-     * @returns {Array<Object>} - [{url: String, number: Number, isVerified: Boolean}]
      */
-    static getStagingDeployCashPRList(issue) {
-        let PRListSection = issue.body.match(/pull requests:\*\*\r?\n((?:-.*\r?\n)+)\r?\n\r?\n?/) || [];
-        if (PRListSection.length !== 2) {
+    static getStagingDeployCashPRList(issue: OctokitIssueItem): StagingDeployCashPR[] {
+        let PRListSection: RegExpMatchArray | null | [] | string = issue.body?.match(/pull requests:\*\*\r?\n((?:-.*\r?\n)+)\r?\n\r?\n?/) ?? [];
+        if (PRListSection?.length !== 2) {
             // No PRs, return an empty array
             console.log('Hmmm...The open StagingDeployCash does not list any pull requests, continuing...');
             return [];
         }
+
         PRListSection = PRListSection[1];
-        const PRList = _.map([...PRListSection.matchAll(new RegExp(`- \\[([ x])] (${PULL_REQUEST_REGEX.source})`, 'g'))], (match) => ({
+        const PRList = [...PRListSection.matchAll(new RegExp(`- \\[([ x])] (${PULL_REQUEST_REGEX.source})`, 'g'))].map((match) => ({
             url: match[2],
             number: Number.parseInt(match[3], 10),
             isVerified: match[1] === 'x',
         }));
-        return _.sortBy(PRList, 'number');
+        // eslint-disable-next-line no-nested-ternary
+        return PRList.sort((a, b) => (a.number > b.number ? 1 : b.number > a.number ? -1 : 0));
     }
 
     /**
      * Parse DeployBlocker section of the StagingDeployCash issue body.
      *
      * @private
-     *
-     * @param {Object} issue
-     * @returns {Array<Object>} - [{URL: String, number: Number, isResolved: Boolean}]
      */
-    static getStagingDeployCashDeployBlockers(issue) {
-        let deployBlockerSection = issue.body.match(/Deploy Blockers:\*\*\r?\n((?:-.*\r?\n)+)/) || [];
+    static getStagingDeployCashDeployBlockers(issue: OctokitIssueItem): StagingDeployCashBlocker[] {
+        let deployBlockerSection: RegExpMatchArray | null | [] | string = issue.body?.match(/Deploy Blockers:\*\*\r?\n((?:-.*\r?\n)+)/) ?? [];
         if (deployBlockerSection.length !== 2) {
             return [];
         }
+
         deployBlockerSection = deployBlockerSection[1];
-        const deployBlockers = _.map([...deployBlockerSection.matchAll(new RegExp(`- \\[([ x])]\\s(${ISSUE_OR_PULL_REQUEST_REGEX.source})`, 'g'))], (match) => ({
+        const deployBlockers = [...deployBlockerSection.matchAll(new RegExp(`- \\[([ x])]\\s(${ISSUE_OR_PULL_REQUEST_REGEX.source})`, 'g'))].map((match) => ({
             url: match[2],
             number: Number.parseInt(match[3], 10),
             isResolved: match[1] === 'x',
         }));
-        return _.sortBy(deployBlockers, 'number');
+
+        return deployBlockers.sort((a, b) => a.number - b.number);
```

Comment:
This sort looks too simplified related to what you did here

```ts
return PRList.sort((a, b) => (a.number > b.number ? 1 : b.number > a.number ? -1 : 0));
```

Reply:
I brought back the old way and replaced ternary operators with if statements so eslint doesn't complain, let me know what you think!